go.mod

```diff
@@ -1,6 +1,6 @@
 module open-cluster-management.io/multicloud-integrations
 
-go 1.19
+go 1.20
```

Comment:
Let's take this chance to upgrade the k8s api to the latest version as well
```
	k8s.io/api 
	k8s.io/apiextensions-apiserver
	k8s.io/apimachinery
	k8s.io/client-go
```

Reply:
I managed to upgrade the packages to their latest patch version, `v0.24.11` but trying to upgrade to the latest minor version `v0.26.2` is causing many more issues with other dependencies. I can keep trying to get them to work or keep them at the latest patch version, what do you think?

Reply:
Got it!  Let's go with the latest patch version 0.24.11.  It is good enough.  The latest minor version may not be stable yet.